ewsonprem_connector.py

```diff
@@ -1509,7 +1509,6 @@ def _decode_uni_string(self, input_str, def_name):
         except Exception as e:
             error_code, error_message = self._base_connector._get_error_message_from_exception(e)
             err = "Error Code: {0}. Error Message: {1}".format(error_code, error_message)
-            self.debug_print("Decoding: {0}. {1}".format(encoded_strings, err))
```

Comment:
Oh, you can print the `err`, but don't print the `ecoded_strings`

Reply:
Oh oops sorry about that. Added the error print back in